ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,63 @@
+<!--
+
+Thank you for taking time to open an issue.
+Please take time to read the instructions, even if you've read it before, as things may have changed in the recent past.
+
+There are three main reasons you may want to open an issue here:
+
+1. a bug report
+2. a feature request
+3. operational issues
```

Comment:
probably we can give people the gist of operational issues here directly?

Reply:
that'd be: 
>  Please check the system status on https://www.traviscistatus.com/ and if you have any questions, please email us at support@travis-ci.com instead.

Reply:
This is mentioned below.